NOTICE

```diff
@@ -0,0 +1,7 @@
+napalm
+Copyright 2015 Spotify AB. All rights reserved.
```

Comment:
This is a matter I thought about it, seems still unclear.
What about adding another line here like:

```
Copyright 2016-2020 NAPALM Automation Team. All rights reserved.
```

Any thoughts @ktbyers @dbarrosop?

Reply:
I am not a lawyer, but my understanding is there should be a single copyright notice only and that identifies the copyright owner at the time the work was first published. The owner should be a legal entity (a person, their employer, an organization, etc.)

Reply:
I am not a lawyer, but my understanding is there should be a single copyright notice only and that identifies the copyright owner at the time the work was first published. The owner should be a legal entity (a person, their employer, an organization, etc.)